<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_superior">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bt_minimizar">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>iconos/minimizar-signo_m.png</normaloff>iconos/minimizar-signo_m.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_maximizar">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>iconos/maximizar_m.png</normaloff>iconos/maximizar_m.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_normal">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>iconos/mini_m.png</normaloff>iconos/mini_m.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_cerrar">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>iconos/marca-x_m.png</normaloff>iconos/marca-x_m.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_contenido">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(176, 18, 235);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QPushButton" name="bt_abrir">
                <property name="styleSheet">
                 <string notr="true">color: rgb(176, 18, 235);
font: 12pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235)</string>
                </property>
                <property name="text">
                 <string>ABRIR MUSICA</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>iconos/musica_m.png</normaloff>iconos/musica_m.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="grafica_layout" stretch="">
                <property name="spacing">
                 <number>6</number>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QSlider" name="slider_tiempo">
                  <property name="styleSheet">
                   <string notr="true">QSlider::groove:horizontal{
	border: 4px solid rgb(176, 18, 235);
	height: 4px;
	background: rgb(176, 18, 235)
}

QSlider::handle:horizontal{
	background: rgb(176, 18, 235);
	margin: -4px
}

QSlider::add-page:horizontal{
	background-color: black;
	border: 2px solid rgb(176, 18, 235);
	border-radius: 3px
}</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="indicador_posicion">
                      <property name="styleSheet">
                       <string notr="true">color: rgb(176, 18, 235);
font: 12pt &quot;Magnum&quot;</string>
                      </property>
                      <property name="text">
                       <string>00:00</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="indicador_tiempo">
                      <property name="styleSheet">
                       <string notr="true">color: rgb(176, 18, 235);
font: 12pt &quot;Magnum&quot;</string>
                      </property>
                      <property name="text">
                       <string>00:00</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <widget class="QPushButton" name="bt_atras">
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>iconos/atras_m.png</normaloff>iconos/atras_m.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="bt_pausa">
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>iconos/pausa_m.png</normaloff>iconos/pausa_m.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="bt_reproducir">
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>iconos/play_m.png</normaloff>iconos/play_m.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="bt_adelante">
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>iconos/siguiente_m.png</normaloff>iconos/siguiente_m.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                         <widget class="QPushButton" name="bt_volumen_0">
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset>
                            <normaloff>iconos/mudo_m.png</normaloff>iconos/mudo_m.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>20</width>
                            <height>20</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="bt_volumen_50">
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset>
                            <normaloff>iconos/bajar-volumen_m.png</normaloff>iconos/bajar-volumen_m.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>20</width>
                            <height>20</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="bt_volumen_100">
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset>
                            <normaloff>iconos/subir-volumen_m.png</normaloff>iconos/subir-volumen_m.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>20</width>
                            <height>20</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                         <widget class="QSlider" name="slider_volumen">
                          <property name="minimumSize">
                           <size>
                            <width>0</width>
                            <height>30</height>
                           </size>
                          </property>
                          <property name="sizeIncrement">
                           <size>
                            <width>0</width>
                            <height>0</height>
                           </size>
                          </property>
                          <property name="styleSheet">
                           <string notr="true">QSlider::groove:horizontal{
	border: 1px solid rgb(176, 18, 235);
	height: 4px;
	background: rgb(176, 18, 235)
}

QSlider::handle:horizontal{
	background: rgb(176, 18, 235);
	
	width: 28px;
	height: 28px;

	left: 11px;
	right: 11px;

	margin: -12px;
	border-radius: 14px
}

QSlider::add-page:horizontal{
	background-color: white;
	border: 1px solid white
}</string>
                          </property>
                          <property name="maximum">
                           <number>100</number>
                          </property>
                          <property name="value">
                           <number>50</number>
                          </property>
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="indicador_volumen">
                          <property name="styleSheet">
                           <string notr="true">color: rgb(176, 18, 235);</string>
                          </property>
                          <property name="text">
                           <string>50</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QListWidget" name="lista_musica">
              <property name="styleSheet">
               <string notr="true">color: rgb(176, 18, 235)</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>15</number>
              </property>
              <property name="bottomMargin">
               <number>8</number>
              </property>
              <item>
               <widget class="QLabel" name="titulo_cancion">
                <property name="styleSheet">
                 <string notr="true">color: rgb(176, 18, 235);</string>
                </property>
                <property name="text">
                 <string>TITULO:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titulo_artista">
                <property name="styleSheet">
                 <string notr="true">color: rgb(176, 18, 235);</string>
                </property>
                <property name="text">
                 <string>ARTISTA:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="genero_cancion">
                <property name="styleSheet">
                 <string notr="true">color: rgb(176, 18, 235);</string>
                </property>
                <property name="text">
                 <string>GENERO:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="annio_cancion">
                <property name="styleSheet">
                 <string notr="true">color: rgb(176, 18, 235);</string>
                </property>
                <property name="text">
                 <string>AÑO:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
